src/shared/components/TopcoderHeader/mobile/Header/index.jsx

```diff
@@ -19,7 +19,11 @@ export default function Header({
   return (
     <div styleName="header-wrapper">
       <div
-        onClick={open}
+        onClick={({ target }) => {
+          if (target.nodeName !== 'A' || !target.getAttribute('href')) {
```

Comment:
@elfman This looks goofy to me. I guess, it is better to add some code to the Log In / Out buttons in question, to ensure that when they are clicked, the events are not propagated to the menu header under them